src/Components/Admins/index.js

```diff
@@ -15,9 +15,10 @@ const Admins = () => {
   const headers = ['Name', 'Last Name', 'Email'];
   const error = useSelector((state) => state.admins.error);
   const dispatch = useDispatch();
+  const token = sessionStorage.getItem('token');
 
   useEffect(() => {
-    dispatch(getAdmins());
+    dispatch(getAdmins(token));
```

Comment:
```suggestion
    dispatch(getAdmins());
```
The token should be set inside the thunk